packages/react-frontend/src/routes/MyReviews.jsx

```diff
@@ -1,11 +1,87 @@
 import Navbar from "./Navbar";
+import AccountCircleIcon from "@mui/icons-material/AccountCircle";
+import { useState, useEffect } from "react";
+import { useParams } from "react-router-dom";
+import Review from "../../components/Review";
+import "./MyReview.css";
+
 export default function MyReviews() {
+  const { username } = useParams();
+  const [user, setUser] = useState(null);
+  const [reviews, setReviews] = useState(null);
+
+  useEffect(() => {
+    async function fetchData() {
+      try {
+        let response = await fetch(
+          `http://localhost:8000/reviews/${username}`
```

Comment:
i've configured the CD so this link is going to have to change to the azure link instead of localhost:8000, so it should be 'jukeboxd-music.azurewebsites.net' from now on, instead. but u can do this after the other changes lol